site/en/blog/topics-enhancements/index.md

```diff
@@ -2,9 +2,9 @@
 layout: 'layouts/blog-post.njk'
 title: Enhancements to the Topics API
 description: >
-  Updates to the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
+ Updates to top topics selection, the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
 subhead: >
-  Updates to the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
+  Updates to top topics selection, the Topics taxonomy and filtering mechanisms, along with speed improvements and enhanced user controls.
```

Comment:
comma here? or no? " ...selection, the Topics taxonomy, ..."

Reply:
I think it's OK as-is, because the sense is "the Topics ... filtering mechanisms".

...but I know it's getting a bit wordy :).